website/docs/d/all_namespaces.html.markdown

```diff
@@ -0,0 +1,29 @@
+---
+layout: "kubernetes"
+page_title: "Kubernetes: kubernetes_all_namespaces"
+sidebar_current: "docs-kubernetes-data-source-all-namespaces"
+description: |-
+  Lists all namespaces within a cluster.
+---
+
+# kubernetes_all_namespaces
+
+This data source provides a mechanism for listing the names of all available namespaces in a Kubernetes cluster.
+It can be used check for existence of a specific namespaces or to apply another resource to all or a subset of existing namespaces in a cluster.
```

Comment:
```suggestion
It can be used to check for existence of a specific namespaces or to apply another resource to all or a subset of existing namespaces in a cluster.
```